Makefile

```diff
@@ -1,4 +1,4 @@
 
 docker-nearcore:
-	DOCKER_BUILDKIT=1 docker build -t nearcore -f ops/Dockerfile .
+	sudo DOCKER_BUILDKIT=1 docker build -t nearcore -f ops/Dockerfile .
```

Comment:
should be sudo make, no? 
my docker on Mac is under user

Reply:
yea.. can change back. i switched because it's easier to see exactly what needs studio without looking in the Makefile/script